File: php/load-data.php
<?php
// php.ini
// extension=php_com_dotnet.dll
$WshShell = new com("WScript.Shell");

$ExcelApp = new com("Excel.Application") or die("Unable to instantiate Excel");

// 表示する
$ExcelApp->Visible = true;
// 警告を出さないようにする
$ExcelApp->DisplayAlerts = false;

// 開く
$workbook = $ExcelApp->Workbooks->Open( getcwd() ."\\syain.xlsx" );
// 最大化
$ExcelApp->ActiveWindow->WindowState = -4137;

// 保存した事にする
$workbook->Saved = true;

// アプリを終了
$ExcelApp->Quit();

// オブジェクト初期化
$ExcelApp = null;

$WshShell = null;
?>

File: python/load-data.py
import win32com.client
import os
from tkinter import messagebox

curdir = os.getcwd()

ExcelApp = win32com.client.Dispatch("Excel.Application")

ExcelApp.Visible = True

ExcelApp.DisplayAlerts = False

workbook = ExcelApp.Workbooks.Open( curdir + "\\syain.xlsx" )

ExcelApp.ActiveWindow.WindowState = -4137

messagebox.showinfo("タイトル", "STOP")

workbook.Saved = True

ExcelApp.Quit()

